### PR TITLE
delete person case v2

### PR DESCRIPTION
**Pull request by czue**
https://trello.com/c/id6say3l/403-ucr-db-scaling

this builds off (and should be merged after) https://github.com/dimagi/commcare-hq/pull/23605

can review by commit.

@emord fyi there are a few more management commands in here that I missed on my first search (think I was searching complete words which missed raw references to the table name)

### CODE REVIEW

File: custom/icds_reports/README.md [hunk 1]
```diff
@@ -159,7 +159,10 @@ Known areas that can be changed to improve performance
 
    The highest ROI are moving reports based on ccs_record_monthly_v2, child_health_monthly_v2 and person_cases_v2 (in that order).
 
-   The end goal being no longer needed either monthly UCR (queries only on the base case UCR & appropriate form UCR) and reducing the number of columns in person_cases_v2
+   The end goal being no longer needed either monthly UCR (queries only on the base case UCR & appropriate form UCR) and reducing the number of columns in person_cases_v2.
+   
+   As of March, 2019 we have rolled out person_cases_v3 which achieves the reduction in columns in person_case_v2.
```
Comment (emord) [inline custom/icds_reports/README.md]: Thanks for updating this. I imagine this section is going to need a big overhaul after citus and other current work completes

**Review by emord — APPROVED** [review]
Assuming we'll keep this around for ~week and then drop the table in a couple weeks? (300 GB drop should be satisfying)

**Comment by czue** [comment]
@emord shall we :ship: this? I think it's probably been enough time.

Also, deleting the data source won't delete the table, right? We'll do that manually at some point in the near future?

**Review by emord — APPROVED** [review]

**Comment by emord** [comment]
The build failed due to migrations that needed to be merged. I went ahead and rebased on master as it looks like that's what you had done a few hours ago

**Comment by czue** [comment]
thanks!

**Pull request merged**